recipe/meta.yaml

```diff
@@ -108,28 +111,29 @@ outputs:
   - name: "muscat-devenv"
     version: {{ version }}
     requirements:
-      - {{ compiler('cxx') }}
-      - python {{ python }}
-      - numpy
-      - sympy
-      - cython
-      - eigency
-      - mkl
-      - mkl-include
-      - setuptools-scm
-      - eigen
-      - scipy >=1.9
-      - h5py
-      - pycgns
-      - networkx >=3.0
-      - libboost-headers
-      - pytest
-      - vtk
-      - tbb
-      - tbb-devel
-      - matplotlib
-      - coverage
-      - pywin32  # [win]
+      run:
+        - {{ compiler('cxx') }}
```

Comment:
This is weird as a runtime dependency

Reply:
the package "muscat-devenv" has all the dependencies to work on the Muscat projet "as a developer". Historically to have all the dependencies of Muscat (without muscat) the anwser was "conda install Muscat; conda uninstall Muscat". The idea here is to install muscat-devenv from conda-forge and Muscat from sources (git clone).

Is this a good idea??

Reply:
Can't say if good or not, but surely an unusual purpose for a conda package published in a community channel. Usually these things are more suitable for an internal channel (e.g. anaconda.org/muscat).

Reply:
e.g. this wouldn't have passed staged-recipes without a good explanation.